proxa/Docusign-Integration
# feat: added basic springboot server with mock data and rest endpoints

+ Added basic template for prototype backend server
+ Added a REST endpoint
+ Created an h2 database schema
+ Added mock data into h2 database

**Test Procedures**
1. checkout branch
2. cd into the server directory
3. run gradle bR
4. When the server is running, hit the endpoint: http://localhost:8080/all
5. You should get back a JSON data structure containing a builder's name and a binary blob of the Lien Waiver PDF
6. Hit the endpoint to search by builder name: http://localhost:8080/builder?builderName=Blake%20Bartenbach
7. Verify you get the same data back
8. Hit the endpoint to search by builder name with someone that doesn't exist: http://localhost:8080/builder?builderName=Joe%20Biden
9. Verify you get an empty JSON array
